Description: Upstream changes introduced in version 2.6-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 ossec-hids (2.6-1) unstable; urgency=low
 .
   * Initial release (Closes: #361954)
 .
 The person named in the Author field signed this changelog entry.
Author: Jose Antonio Quevedo Muñoz <joseantonio.quevedo@gmail.com>
Bug-Debian: http://bugs.debian.org/361954

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- ossec-hids-2.6.orig/src/Config.Make
+++ ossec-hids-2.6/src/Config.Make
@@ -22,6 +22,6 @@ OS_CRYPTO = ${PT}os_crypto/os_crypto.a
 OS_SHARED = ${PT}shared/lib_shared.a
 OS_ROOTCHECK = ${PT}rootcheck/rootcheck_lib.a
 OS_CONFIG = ${PT}config/lib_config.a
-OS_ZLIB = ${PT}os_zlib/os_zlib.c ${PT}external/libz.a
-OS_LINK = ${IEXTRA}
+OS_ZLIB = ${PT}os_zlib/os_zlib.c
+OS_LINK = ${IEXTRA} -I/usr/include -lz
 DBS = cdb/cdb.a
--- ossec-hids-2.6.orig/src/Makeall
+++ ossec-hids-2.6/src/Makeall
@@ -129,10 +129,10 @@ fi
 # Getting values for each action
 if [ "X${ARGV}" = "Xall" ]; then
     DIRECTORIES="${LIBS} ${SOURCES} ${BINARIES}" 
-    DOZLIB="x"
+    DOZLIB=""
 elif [ "X${ARGV}" = "Xlibs" ]; then
     DIRECTORIES="${LIBS} ${SOURCES}"
-    DOZLIB="x"
+    DOZLIB=""
 elif [ "X${ARGV}" = "Xrootcheck" ]; then
     DIRECTORIES="os_xml os_regex os_net ${SOURCES} ${ROOTCHECKBIN}"    
 elif [ "X${ARGV}" = "Xbuild" ]; then
